addons/html_editor/tests/test_controller.py

```diff
@@ -183,24 +194,25 @@ def test_05_internal_link_preview(self):
         self.assertTrue('error_msg' in response_wrong_record.text)
 
         # retrieve metadata of a url not directing to a record
-        response_not_record = self.url_open(
-            '/html_editor/link_preview_internal',
-            data=json_safe.dumps({
-                "params": {
-                    "preview_url": "/odoo/users",
-                }
-            }),
-            headers=self.headers
-        )
-        self.assertEqual(200, response_not_record.status_code)
-        self.assertTrue('other_error_msg' in response_not_record.text)
```

Comment:
Why we removed this? don't you think if it is not a record or not a page then it should give other_error_msg?